### ISSUE TITLE
Range dialog

### ISSUE THREAD

**Issue by mtrakal** [code-troopers/android-betterpickers#293]
It would be nice when betterpickers lib can integrate function of MaterialDateRangePicker with native betterpickers design, because MaterialDateRangePicker uses a little different design of Date dialog and it does not look consistent in apps.

https://github.com/borax12/MaterialDateRangePicker

**Comment by fchauveau** [comment]
Yes, but it a big evolution, and we do not have such time for doing this now
